static/app/components/replays/replayTagsTableRow.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+import {OrganizationContext} from 'sentry/views/organizationContext';
+import {RouteContext} from 'sentry/views/routeContext';
+import ReplayTagsTableRow from './replayTagsTableRow';
+
+const tags = [
+  {
+    key: 'foo',
+    value: ['bar', 'baz'],
+  },
+  {
+    key: 'release',
+    value: ['1.0.0', '2.0.0'],
+  },
+];
+
+function TestComponent({children}) {
+  const {organization, router} = initializeOrg();
+
+  return (
+    <OrganizationContext.Provider value={organization}>
+      <RouteContext.Provider
```

Comment:
Do we need this at all? It doesn't look like we use org/routes in `ReplayTagsTableRow`?

Reply:
It seems like we use it somewhere in the tree, when I try to run the tests without it I get:


![SCR-20220913-g0j](https://user-images.githubusercontent.com/39612839/189944460-2a64330e-0a23-4a5d-abe0-3f1c54bc2b29.png)